biz.aQute.bnd/src/aQute/bnd/main/bnd.java

```diff
@@ -932,7 +932,13 @@ public void perProject(ProjectWorkspaceOptions opts, PerProject run) throws Exce
 		final Set<Project> projectsWellDone = new HashSet<>();
 
 		for (Project p : projects) {
-			final Set<Project> projectDeps = p.getDependents(); // is ordered
+			final Collection<Project> projectDeps = p.getDependson(); // ordered
+			if (opts.verbose()) {
+				System.out.println("Project dependencies for: " + p.getName());
+				projectDeps.stream()
+					.forEachOrdered(pr -> System.out.println(
```

Comment:
LOL. I did not notice this before, but since all you want to do with the collection is `forEach`, you don't even need `stream()`. In fact it is a waste to make a stream for this. Just use `projectDeps.forEach` since Collection now has `forEach`.